android/src/main/kotlin/com/amplitude/amplitude_flutter/FlutterLibraryPlugin.kt

```diff
@@ -9,7 +9,11 @@ class FlutterLibraryPlugin(val library: String): Plugin {
     override lateinit var amplitude: Amplitude
 
     override fun execute(event: BaseEvent): BaseEvent? {
-        event.library = library
+        if (event.library == null) {
```

Comment:
Shouldn't here be `!= null`?

Reply:
We can only merge the `event.library` value if its not null. If its null, we just want to use the flutter library version `Amplitude-Flutter/4.0.0`.

Reply:
It seems to be redundant as the library parameter is a string type value in the primary constructor and is declared as a non-nullable type (String), it cannot be null.

Reply:
`event.library` can be null. I was thinking about `library` of the plugin class.